cli/dtaas.toml

```diff
@@ -1,18 +1,18 @@
 # This is the config for DTaaS CLI
 
 name = "Digital Twin as a Service (DTaaS)"
-version = "0.1.0"
+version = "0.1.1"
 owner = "The INTO-CPS-Association"
 git-repo = "https://github.com/into-cps-association/DTaaS.git"
 
 [common]
 # absolute path to the DTaaS application directory
 server-dns = "localhost"
-path = "/home/Desktop/DTaaS"
+path = "C:\\Users\\Mohamed\\DTaaS"
```

Comment:
please revert this line.
Do add a comment before this line mentioning the need for double backward slashes for Windows paths.